db/seeds.rb

```diff
@@ -1,4 +1,20 @@
 # Seed the DB with an admin user and a few version records
+if Archiver.allowed_hosts.empty?
+  puts <<~WARN
+    No Archiver allowed hosts configured. This will result in a large download.
+    You may set this with the ALLOWED_ARCHIVE_HOSTS environment variable (See .env.example).
```

Comment:
This is awesome, thank you!

That said, I wonder if we could explain more here, e.g:

> You don't have any allowed hosts configured for Archiver! That means every version of every web page in the seed file will be downloaded to your computer, which could take a while. To avoid this, stop this script and set the ALLOWED_ARCHIVE_HOSTS environment variable (see .env.example).
> 
> (Allowed hosts are hosts that are allowed to remotely store the raw content of a version of a web page on behalf of your database. If a version is stored at an allowed host instead of locally, you'll need to be online in order to view the webpage itself or diffs between version. Metadata is still stored locally in your database for offline use.)

Do you think that’s helpful or just way too verbose? (What you’ve already got is good, too.)

Reply:
I think we can put that in the `.env.example` file and really emphasize that someone should go look at it for more information.